tanker-identity.gemspec

```diff
@@ -21,13 +21,14 @@ Gem::Specification.new do |spec|
 
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "rbnacl", "~> 7.0"
+  spec.add_dependency "base64", "~> 0.2.0"
```

Comment:
Note: base64 was change from "default" to "bundled" gem in Ruby 3.4, which means it can now be uninstalled from a default Ruby install. As such, it needs to be explicitly listed as a dependency now.